testing-guidance/workflow-tests/physics/physics-lkg-workflows.md

```diff
@@ -135,33 +231,97 @@ Any project that has the following Gems enabled:
 |---------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Changing Rigid Body Properties in Workflows | <ol><li>In any of the workflows with a Rigid Body found on this page change the rigid body properties for the scenario under test.</li><li>Run the Simulation in Editor Game Mode or in the Game Launcher.</li></ol> | <ul><li>Scene behaves according to the the Ridgid Body changes made.</li><li>Extra care should be tested around the following Rigid Body Properties:</li><ul><li>Angular & Linear damping</li><li>COM</li><li>Kinematic Mode</li><li>Mass</li><li>Angular Velocity</li><li>Compute Intertia</li></ul></ul> |
+| Changing Rigid Body Properties in Workflows | <ol><li>In any of the workflows with a Rigid Body found on this page change the rigid body properties for the scenario under test.</li><li>Change System and Scene Configuration settings in PhysX Configuration (PREVIEW) tool and verify that provided changes are reflected in rigid bodies behavior during simulation.</li><li>Enable CCD in PhysX Configuration (PREVIEW) tool and verify that rigid bodies with CCD enabled have better hit detection during fast collision scenarios.</li><li>Enable/Disable gravity enabled toggle in PhysX Rigid Body.</li><li>Enable/Disable start asleep toggle in PhysX Rigid Body.</li><li>Set sleep threshold for PhysX Rigid body and verify that entity goes to sleep when applied forces are lower than set in threshold.</li><li>Run the Simulation in Editor Game Mode or in the Game Launcher.</li></ol> | <ul><li>Scene behaves according to the the Ridgid Body changes made.</li><li>Time step and gravity values set in PhysX Configuration (PREVIEW) tool correctly impact behaviors of rigid bodies in a scene.</li><li>Rigid bodies with CCD enabled have better hit detection during scenarios involving big velocities.</li><li>Rigid bodies with gravity enabled toggle turned off do not react to gravity set in PhysX Configuration (PREVIEW)</li><li>Rigid bodies with start asleep toggle turned on have to be woken up by a force to start simulating.</li><li>Rigid bodies with set threshold go to sleep when applied forces are lower than set threshold.</li><li>Extra care should be tested around the following Rigid Body Properties:</li><ul><li>Angular & Linear damping</li><li>COM</li><li>Kinematic Mode</li><li>Mass</li><li>Angular Velocity</li><li>Compute Intertia</li></ul></ul> |
```

Comment:
Step 2, 3, 6: Move verification language to things to watch out for and add in language for actionable steps to achieve that verification.